modules/Makefile.circleci-2.0

```diff
@@ -62,7 +62,7 @@ circle\:deploy-kubernetes:
 ## Cleanup docker images on CircleCI
 circle\:cleanup-docker:
 	@echo "INFO: Cleaning up older dangling docker images"
-	@docker images -f "dangling=true" --format "{{.ID}} {{.CreatedSince}}" | grep "weeks ago" | cut -f 1 -d " " | xargs --no-run-if-empty docker rmi || true
+	@docker images -f "dangling=true" --format "{{.ID}} {{.CreatedSince}}" | grep "days\|weeks" | cut -f 1 -d " " | xargs --no-run-if-empty docker rmi || true
```

Comment:
Why not allow repositories to specify the retention period? You can define a variable with a default value, and repos needing less or more can override.